packages/gravity-ui-nunjucks/src/_patterns/00-particles/01-typography/01-text-sizes.njk

```diff
@@ -0,0 +1,7 @@
+<dl>
+{% for textSize in textSizes %}
+  <dt><strong>CSS util class</strong>: <code>.grav-u-font-size-{{ textSize.classSuffix }}</code></dt>
+  <dt><strong>SASS mixin</strong>: <code>@include grav-font-size({{ textSize.scale }});</code></dt>
```

Comment:
Same question about gravy as the sass mixin

Reply:
No Gravy here. This is `grav-...` (no `y`) :-)